pkg/util/paramtable/component_param.go

```diff
@@ -3094,11 +3095,20 @@ user-task-polling:
 		Key:          "queryNode.queryStreamBatchSize",
 		Version:      "2.4.1",
 		DefaultValue: "4194304",
-		Doc:          "return batch size of stream query",
+		Doc:          "return min batch size of stream query",
```

Comment:
Can we validate if the min size is less than max? What is the consequence if they are configured wrong?

Reply:
We will flush till buffer size large than min, then split message size less than max.
If max less than min, we will split all message.